lib/growth/load_reference.ex

```diff
@@ -15,7 +15,7 @@ defmodule Growth.LoadReference do
         {:error, "Invalid data type: #{data_type}"}
 
       file_name ->
-        file_path = "priv/indicators/" <> file_name
+        file_path = Application.app_dir(:growth, "priv/indicators") <> "/#{file_name}"
```

Comment:
Probably could use [`Path.join/2`](https://hexdocs.pm/elixir/1.17.3/Path.html#join/2) to merge the different parts in a safely manner.